_hardcoded/tracing.go

```diff
@@ -110,37 +110,52 @@ func SetupGlobalTraceProviderAndExporterForTest() (*tracetest.InMemoryExporter,
 // Right now we only support logging IDs in the format that Datadog expects.
 func MuxServerMiddleware(serviceName string) func(http.Handler) http.Handler {
 	otlmux := otelmux.Middleware(serviceName, otelmux.WithPropagators(otel.GetTextMapPropagator()))
+	fmt.Println("Adding mux server middleware")
```

Comment:
Is this intentional?

Reply:
Nope, removed in a prior cleanup, but this code change came from a real client that I was testing. Thanks for catching that.